src/pretalx/submission/models/cfp.py

```diff
@@ -48,7 +48,10 @@ def default_fields():
         "do_not_record": {"visibility": "optional"},
         "image": {"visibility": "optional"},
         "track": {"visibility": "do_not_ask"},
-        "duration": {"visibility": "do_not_ask"},
+        "duration": {
+            "visibility": "do_not_ask",
+            "required_duration": True,
```

Comment:
This would simultaneously make the duration field **not** visible and **required**, which I don't think is sensible? Depending on what you meant to do, you should change the `visibility` to `optional` or `required` – but, crucially, I don't understand why you are changing the default visibility of the duration field at all. Could you please explain your reasoning behind this change?